pymaid/pymaid.py

```diff
@@ -1164,63 +1165,105 @@ def get_connectors ( skids, remote_instance = None, incoming_synapses = True, ou
     for a in range( 0, len(skids), 50 ):
         for i,s in enumerate(skids[ a:a+50] ):
-            tag = 'skeleton_ids[%i]' % i 
+            tag = 'skeleton_ids[%i]' % i
             get_connectors_GET_data[tag] = str( s )
 
         if incoming_synapses is True:
             get_connectors_GET_data['relation_type']='presynaptic_to'
             remote_get_connectors_url = remote_instance.get_connectors_url( project_id ) + '?%s' % urllib.parse.urlencode(get_connectors_GET_data)
             cn_data += [ e + ['presynaptic_to'] for e in remote_instance.fetch( remote_get_connectors_url )['links'] ]
 
+            #Add tags to tags dict
+            tags_dict = remote_instance.fetch( remote_get_connectors_url )['tags']
+
         if outgoing_synapses is True:
             get_connectors_GET_data['relation_type']='postsynaptic_to'
             remote_get_connectors_url = remote_instance.get_connectors_url( project_id ) + '?%s' % urllib.parse.urlencode(get_connectors_GET_data)
             cn_data += [ e + ['postsynaptic_to'] for e in remote_instance.fetch( remote_get_connectors_url )['links'] ]
 
+            #Add tags to tags dict
+            tags_dict = dict(tags_dict, **remote_instance.fetch( remote_get_connectors_url )['tags'])
+
         if abutting is True:
             get_connectors_GET_data['relation_type']='abutting'
             remote_get_connectors_url = remote_instance.get_connectors_url( project_id ) + '?%s' % urllib.parse.urlencode(get_connectors_GET_data)
             cn_data += [ e + ['abutting'] for e in remote_instance.fetch( remote_get_connectors_url )['links'] ]
 
+            #Add tags to tags dict
+            tags_dict = dict(tags_dict, **remote_instance.fetch( remote_get_connectors_url )['tags'])
+
         if gap_junctions is True:
             get_connectors_GET_data['relation_type']='gapjunction_with'
             remote_get_connectors_url = remote_instance.get_connectors_url( project_id ) + '?%s' % urllib.parse.urlencode(get_connectors_GET_data)
-            cn_data += [ e + ['gap_junction'] for e in remote_instance.fetch( remote_get_connectors_url )['links'] ]              
+            cn_data += [ e + ['gap_junction'] for e in remote_instance.fetch( remote_get_connectors_url )['links'] ]
 
-    df = pd.DataFrame(  cn_data,
-                        columns = [ 'skeleton_id', 'connector_id', 'x', 'y', 'z', 'confidence', 'creator_id', 'treenode_id', 'creation_time', 'edition_time' , 'type' ],
-                        dtype=object
-                         )
+            #Add tags to tags dict
+            tags_dict = dict(tags_dict, **remote_instance.fetch( remote_get_connectors_url )['tags'])
+
+
+        #Convert tag_flag to 0 or 1 if necessary
+        if type(tag_flag) != type( int() ):
+            tag_flag = int(tag_flag)
+
+        if type(tag_flag) != type( bool() ):
+                tag_flag = tag_flag == 1
+

```

Comment:
Converting is not necessary if you just use
```
if tag_flag:
   for cn in range( len ( cn_data  ) ):
....
```
This treats 0/False and 1/True the same.